README.md

```diff
@@ -30,14 +30,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+DOM stands for Document Object Model. It is a model of the objects in an html or xml document. It provides an interface (API) between what the browser renders and the document allowing for it to be manipulated.
```

Comment:
Great wording.  You may also want to say that it allows a program to read and manipulate the page’s content, structure, and styles.